src/js/stores/SessionStore.js

```diff
@@ -11,7 +11,7 @@ var _session = {
 	emSize: 10,
 	width: 640,
 	timerOn: (localStorage.hasOwnProperty("model") === true),
-	nowOffset: getTZOffset(now),
+	nowOffset: now.getTimezoneOffset(),
```

Comment:
Actually this line I am not sure about. I couldn't find getTZOffset() anywhere in the code base -- is this an equivalent replacement? Perhaps not, because I don't think the "your current time zone" format looks right after this change.